src/main/java/io/swagger/codegen/languages/php/AbstractPhpCodegen.java

```diff
@@ -0,0 +1,667 @@
+        } else if (schema instanceof DateTimeSchema) {
+            // TODO
+        } else if (schema instanceof NumberSchema) {
+            NumberSchema numberSchema = (NumberSchema) schema;
+            if (
+                    numberSchema.getDefault() != null
+                            && (SchemaTypeUtil.FLOAT_FORMAT.equals(schema.getFormat()) || SchemaTypeUtil.DOUBLE_FORMAT.equals(schema.getFormat()))
+                    ) {
+                return numberSchema.getDefault().toString();
+            }
+        } else if (schema instanceof IntegerSchema) {
+            IntegerSchema integerSchema = (IntegerSchema) schema;
+            if (integerSchema.getDefault() != null) {
+                return integerSchema.getDefault().toString();
+            }
+        }
+
+        return null;
+    }
+
+    @Override
+    public void setParameterExampleValue(CodegenParameter p) {
+        String example;
+
+        if (p.defaultValue == null) {
+            example = p.example;
+        } else {
+            example = p.defaultValue;
+        }
+
+        String type = p.baseType;
+        if (type == null) {
+            type = p.dataType;
+        }
+
+        if ("String".equalsIgnoreCase(type)) {
+            if (example == null) {
+                example = p.paramName + "_example";
+            }
+            example = "\"" + escapeText(example) + "\"";
+        } else if ("Integer".equals(type) || "int".equals(type)) {
+            if (example == null) {
+                example = "56";
+            }
+        } else if ("Float".equalsIgnoreCase(type) || "Double".equalsIgnoreCase(type)) {
+            if (example == null) {
+                example = "3.4";
+            }
+        } else if ("BOOLEAN".equalsIgnoreCase(type) || "bool".equalsIgnoreCase(type)) {
+            if (example == null) {
+                example = "True";
+            }
+        } else if ("\\SplFileObject".equalsIgnoreCase(type)) {
+            if (example == null) {
+                example = "/path/to/file";
+            }
+            example = "\"" + escapeText(example) + "\"";
+        } else if ("Date".equalsIgnoreCase(type)) {
+            if (example == null) {
+                example = "2013-10-20";
+            }
+            example = "new \\DateTime(\"" + escapeText(example) + "\")";
+        } else if ("DateTime".equalsIgnoreCase(type)) {
+            if (example == null) {
+                example = "2013-10-20T19:20:30+01:00";
+            }
+            example = "new \\DateTime(\"" + escapeText(example) + "\")";
+        } else if (!languageSpecificPrimitives.contains(type)) {
+            // type is a model class, e.g. User
+            example = "new " + getTypeDeclaration(type) + "()";
+        } else {
+            LOGGER.warn("Type " + type + " not handled properly in setParameterExampleValue");
+        }
+
+        if (example == null) {
+            example = "NULL";
+        } else if (Boolean.TRUE.equals(p.isListContainer)) {
```

Comment:
This produces a compile error by `p.isListContainer`.

Where did you get this file? [PhpClientCodegen](https://github.com/swagger-api/swagger-codegen/blob/3.0.0/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PhpClientCodegen.java) on the `3.0.0` branch in the `swagger-codegen` has no copile error. You should have took this as base.